### ISSUE TITLE
Puppet task docker::swarm_token fails, because of frozen string

### ISSUE THREAD

**Issue by khaefeli** [puppetlabs/puppetlabs-docker#432]
## What you expected to happen?

Puppet Task runs and outputs swarm token.

## What happened?
`choria.tasks/task-error: The task errored with a code 1
   {"_output":""}
   /opt/puppetlabs/mcollective/tasks-cache/d8bd4ce747fc489e9e7b19cac9776cfde572cfde54813968f4362b5375e19601/swarm_token.rb:10:in swarm_token: can't modify frozen String (FrozenError)
	from /opt/puppetlabs/mcollective/tasks-cache/d8bd4ce747fc489e9e7b19cac9776cfde572cfde54813968f4362b5375e19601/swarm_token.rb:21:in <main>`

## How to reproduce it?
`mco tasks run docker::swarm_token --node_role manager -I yournode`

## Anything else we need to know?
https://github.com/puppetlabs/puppetlabs-docker/blob/master/tasks/swarm_token.rb#L2
https://github.com/puppetlabs/puppetlabs-docker/blob/master/tasks/swarm_token.rb#L10
If you create a new instance of the string, it works as expected. (use `+=` instead of `<<`)

## Versions:
```
$ /opt/puppetlabs/bin/puppet --version
6.2.0

$ docker --version
Docker version 18.03.1-ce, build 9ee9f40

$ facter os
{
  architecture => "amd64",
  distro => {
    codename => "stretch",
    description => "Debian GNU/Linux 9.7 (stretch)",
    id => "Debian",
    release => {
      full => "9.7",
      major => "9",
      minor => "7"
    }
  },
  family => "Debian",
  hardware => "x86_64",
  name => "Debian",
  release => {
    full => "9.7",
    major => "9",
    minor => "7"
  },
  selinux => {
    enabled => false
  }
}
root

```

## Logs:
```
Paste any relevant logs from a puppet run
```

**Comment by davejrt** [comment]
Hi, we are always happy to take PR's on this module

Would you like to submit a PR with the fix you've identified?

**Comment by khaefeli** [comment]
@davejrt sure, done

**Comment by davejrt** [comment]
fixed in #434

**Comment by khaefeli** [comment]
@davejohnston looks like I forgot the other tasks with the same problem. PR opened
